resources/views/inputs/includes/input-label.blade.php

```diff
@@ -1,11 +1,13 @@
 @props([
     'name',
     'errors',
-    'id'             => null,
-    'label'          => null,
-    'tooltip'        => null,
-    'required'       => false,
-    'auxiliaryTitle' => '',
+    'id'               => null,
+    'label'            => null,
+    'tooltip'          => null,
+    'tooltipClass'     => 'input-tooltip',
```

Comment:
@Highjhacker what additional styling does this class bring?

Reply:
Nothing, removing